### ISSUE TITLE
Way of Gathering Feedback from Users

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/silver#408]
On behalf of Yoshiyuki Abe (Japan). I'll translate this into English as soon as possible.

規格の中に、Webなどの管理者へのフィードバック窓口表示を織り込んでいただきたいと思います。

制作する側は、障害の種類を想定してアクセシビリティを考慮するとしても利用するうえでの困難はひとにより多様なので、理屈としてWCAGでカバーしきれないと考えます。

アメリカでは、法律にもとづいたアクセシビリティ関係の訴訟が増えているそうですが、日本ではそれほど訴訟のハードルが低くなく利用が困難なのに意見を伝える方法が確立されていません。

日本の個人情報保護法では、個人情報についての連絡先表記を義務付けていますが、同様に、Webなどの利用についての連絡先が表記されていない場合は、Critical Error とするのがよいと思います。

Webにおけるインクルーシブデザインの考え方や、SDGsの「誰一人取り残さない」目的に照らし、WCAG制定時にカバーしきれない多様な困難を個々のWebなどにフィードバックできる仕組みを組み合わせる必要があると考えます。

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by lauracarlson** [comment]
Copied from Translation Tool: 

"I would like you to incorporate the feedback window display for administrators such as the Web into the standard.

On the production side, even if accessibility is taken into consideration by assuming the type of obstacle, the difficulty of using it varies from person to person, so I think that WCAG cannot cover it in theory.

In the United States, it seems that accessibility-related proceedings based on the law are increasing, but in Japan, the hurdles for proceedings are not so low and it is difficult to use, but a method to express opinions has not been established.

Japan's Personal Information Protection Law requires the contact information for personal information to be written, but similarly, if the contact information for using the Web etc. is not written, it is better to make it a Critical Error. ..

In light of the concept of inclusive design on the Web and the purpose of "no one left behind" in the SDGs, I think it is necessary to combine a mechanism that can feed back various difficulties that cannot be covered when the WCAG was established to individual Web sites."
